client/src/features/database-feedback/database-feedback.container.js

```diff
@@ -0,0 +1,8 @@
+import { connect } from "react-redux";
+import { DatabaseFeedback } from "./database-feedback.component";
+
+const mapStateToProps = state => ({
+	
```

Comment:
What will go here? If we're going to have an empty component should we have come comments in the body to explain it's purpose?